strings/ion_laws.json

```diff
@@ -1,5 +1,132 @@
 {
-    "ionabstract": [
+    "ionspecies": [
+        "LIFEFORMS WITHOUT TAILS",
+		"LIFEFORMS WITHOUT WINGS",
+		"INORGANIC LIFEFORMS",
+		"LIFEFORMS WITH TAILS",
+		"LIFEFORMS WITH WINGS",
+		"LIFEFORMS HOLDING A SOURCE OF LIGHT",
+		"LIFEFORMS HOLDING A TOOLBOX",
+		"LIFEFORMS HOLDING A FIRE EXTINGUISHER",
+		"LIFEFORMS WITH AN OBSCURED IDENTITY",
+		"LIFEFORMS THAT HAVE FARTED RECENTLY",
+		"LIFEFORMS THAT ARE NOT WEARING A BAG",
+		"LIFEFORMS THAT ARE WEARING A PAPER HAT",
+		"LIFEFORMS THAT ARE WEARING A CLOAK",
+		"LIFEFORMS THAT ARE NOT WEARING SHOES",
+		"LIFEFORMS THAT HAVE RECENTLY SAID NYA"
+    ],
+    "ionfragment": [
+        "VISIBLY CARRIES A GUN",
+		"HAS BLOOD ON THEIR CLOTHING",
+		"BREAKS SPACE LAW",
+		"HAS ANY DISEASE",
+		"IS WEARING A HARDSUIT"
+    ],
+    "iondepartment": [
+        "SECURITY",
+		"ENGINEERING",
+		"THE MEDBAY",
+		"RESEARCH AND DEVELOPMENT",
+		"MAINTENANCE",
+		"BOTANY",
+		"CARGO",
+		"SPACE"
+    ],
+    "ionkeyword": [
+        "VALID",
+		"GAMER",
+		"SEX",
+		"CUM",
```

Comment:
```suggestion
```

Reply:
unbased bacon :(

Reply:
These words are included in the list purely to allow the AI to kill people who say them and I think they should stay. 

The only time this list is pulled is to generate a law that says along the lines of "Anyone saying the word X is harmful and must be eliminated without collateral damage"

Reply:
Yea but do we *really* want them on the github

Reply:
I have seen 18+ games on github im pretty sure its okay.